From cdb616817aee668c81ceb2ac724d20baa97c5154 Mon Sep 17 00:00:00 2001
From: Gareth Kitchen <gkitchen@cookability.biz>
Date: Tue, 17 Jul 2018 09:58:29 +0100
Subject: [PATCH 2/5] Localise NTP to improve GPS TTFF

Testing response times to time.android.com from around the globe reveals
in ms:-

Europe			 <30
Middle East		 <68
North America		<150
Johannesburg		 183
Buenos Aires		 220
Tokyo			 226
Sydney			 276
Hong Kong		 285
Brisbane		 295
Mumbai			 349
Beijing			4691
Shanghai		4906
Russia			 n/a

Whilst time.android.com is NOT used for GPS NTP, North American time servers
are, by specifying north-america.pool.ntp.org as default in the framework,
to align with pixel devices.  I am assuming similar response times to these
servers from around the world.

Great for North America and it appears Europe but it does not address the
global issue. Also, the pool.ntp.org project forbids both hardware and
software vendors from using these default zone names.
http://www.pool.ntp.org/en/vendors.html

It makes sense, therefore, to leverage the ntp.org's existing 'android' vendor
name to make the default ntp server for GPS purposes:
1.android.pool.ntp.org this will return a random but accurate NTP server in
close geopraphic proximity to the device.

Testing on my own build in the UK seems to improve hot and cold TTFF
considerably.

Change-Id: I144af45757efa35b32daf034eece6e046d2bde79
---
 gps/etc/gps.conf                                       | 10 ++--------
 .../base/core/res/res/values-mcc302-mnc220/config.xml  |  1 -
 .../base/core/res/res/values-mcc302-mnc610/config.xml  |  1 -
 .../base/core/res/res/values-mcc310-mnc004/config.xml  |  1 -
 .../base/core/res/res/values-mcc310-mnc120/config.xml  |  1 -
 .../base/core/res/res/values-mcc310-mnc260/config.xml  |  1 -
 .../base/core/res/res/values-mcc310-mnc410/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc220/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc221/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc222/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc223/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc224/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc225/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc226/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc227/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc228/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc229/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc480/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc490/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc580/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc581/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc582/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc583/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc584/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc585/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc586/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc587/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc588/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc589/config.xml  |  1 -
 .../base/core/res/res/values-mcc311-mnc870/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc840/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc854/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc855/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc856/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc857/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc858/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc859/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc860/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc861/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc862/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc863/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc864/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc865/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc866/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc867/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc868/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc869/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc870/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc871/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc872/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc873/config.xml  |  1 -
 .../base/core/res/res/values-mcc405-mnc874/config.xml  |  1 -
 .../base/core/res/res/values-mcc460/config.xml         |  1 -
 overlay/frameworks/base/core/res/res/values/config.xml |  1 -
 54 files changed, 2 insertions(+), 61 deletions(-)

diff --git a/gps/etc/gps.conf b/gps/etc/gps.conf
index 8834810..e6646a8 100644
--- a/gps/etc/gps.conf
+++ b/gps/etc/gps.conf
@@ -14,14 +14,8 @@ XTRA_VERSION_CHECK=1
 # _CLEAR = 0
 ERR_ESTIMATE=0
 
-#Test
-# NTP_SERVER=time.izatcloud.net
-#Asia
-# NTP_SERVER=asia.pool.ntp.org
-#Europe
-# NTP_SERVER=europe.pool.ntp.org
-#North America
-#NTP_SERVER=north-america.pool.ntp.org
+# Worldwide
+NTP_SERVER=1.android.pool.ntp.org
 
 # DEBUG LEVELS: 0 - none, 1 - Error, 2 - Warning, 3 - Info
 #               4 - Debug, 5 - Verbose
diff --git a/overlay/frameworks/base/core/res/res/values-mcc302-mnc220/config.xml b/overlay/frameworks/base/core/res/res/values-mcc302-mnc220/config.xml
index 021547e..539bf06 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc302-mnc220/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc302-mnc220/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.telusmobility.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc302-mnc610/config.xml b/overlay/frameworks/base/core/res/res/values-mcc302-mnc610/config.xml
index 3e39eea..dd3951f 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc302-mnc610/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc302-mnc610/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc310-mnc004/config.xml b/overlay/frameworks/base/core/res/res/values-mcc310-mnc004/config.xml
index 9100472..aef87f8 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc310-mnc004/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc310-mnc004/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=1</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc310-mnc120/config.xml b/overlay/frameworks/base/core/res/res/values-mcc310-mnc120/config.xml
index ad4d194..31d3c6a 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc310-mnc120/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc310-mnc120/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc310-mnc260/config.xml b/overlay/frameworks/base/core/res/res/values-mcc310-mnc260/config.xml
index 993c581..f70595b 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc310-mnc260/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc310-mnc260/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc310-mnc410/config.xml b/overlay/frameworks/base/core/res/res/values-mcc310-mnc410/config.xml
index 7686b46..a1846a2 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc310-mnc410/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc310-mnc410/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc220/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc220/config.xml
index dc69c84..063f029 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc220/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc220/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc221/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc221/config.xml
index dc69c84..063f029 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc221/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc221/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc222/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc222/config.xml
index dc69c84..063f029 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc222/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc222/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc223/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc223/config.xml
index dc69c84..063f029 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc223/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc223/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc224/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc224/config.xml
index dc69c84..063f029 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc224/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc224/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc225/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc225/config.xml
index dc69c84..063f029 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc225/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc225/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc226/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc226/config.xml
index dc69c84..063f029 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc226/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc226/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc227/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc227/config.xml
index dc69c84..063f029 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc227/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc227/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc228/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc228/config.xml
index dc69c84..063f029 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc228/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc228/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc229/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc229/config.xml
index dc69c84..063f029 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc229/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc229/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc480/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc480/config.xml
index 9100472..aef87f8 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc480/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc480/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=1</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc490/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc490/config.xml
index ad4d194..31d3c6a 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc490/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc490/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc580/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc580/config.xml
index dc69c84..063f029 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc580/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc580/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc581/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc581/config.xml
index dc69c84..063f029 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc581/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc581/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc582/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc582/config.xml
index dc69c84..063f029 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc582/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc582/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc583/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc583/config.xml
index dc69c84..063f029 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc583/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc583/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc584/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc584/config.xml
index dc69c84..063f029 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc584/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc584/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc585/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc585/config.xml
index dc69c84..063f029 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc585/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc585/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc586/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc586/config.xml
index dc69c84..063f029 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc586/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc586/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc587/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc587/config.xml
index dc69c84..063f029 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc587/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc587/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc588/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc588/config.xml
index dc69c84..063f029 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc588/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc588/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc589/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc589/config.xml
index dc69c84..063f029 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc589/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc589/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc311-mnc870/config.xml b/overlay/frameworks/base/core/res/res/values-mcc311-mnc870/config.xml
index ad4d194..31d3c6a 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc311-mnc870/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc311-mnc870/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc840/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc840/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc840/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc840/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc854/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc854/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc854/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc854/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc855/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc855/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc855/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc855/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc856/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc856/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc856/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc856/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc857/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc857/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc857/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc857/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc858/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc858/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc858/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc858/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc859/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc859/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc859/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc859/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc860/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc860/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc860/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc860/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc861/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc861/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc861/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc861/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc862/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc862/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc862/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc862/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc863/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc863/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc863/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc863/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc864/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc864/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc864/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc864/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc865/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc865/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc865/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc865/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc866/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc866/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc866/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc866/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc867/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc867/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc867/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc867/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc868/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc868/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc868/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc868/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc869/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc869/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc869/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc869/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc870/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc870/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc870/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc870/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc871/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc871/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc871/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc871/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc872/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc872/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc872/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc872/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc873/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc873/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc873/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc873/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc405-mnc874/config.xml b/overlay/frameworks/base/core/res/res/values-mcc405-mnc874/config.xml
index c13d2a0..542fb87 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc405-mnc874/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc405-mnc874/config.xml
@@ -25,7 +25,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values-mcc460/config.xml b/overlay/frameworks/base/core/res/res/values-mcc460/config.xml
index 3a2cbc6..46223bd 100644
--- a/overlay/frameworks/base/core/res/res/values-mcc460/config.xml
+++ b/overlay/frameworks/base/core/res/res/values-mcc460/config.xml
@@ -26,7 +26,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=asia.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.nokia.com</item>
         <item>SUPL_PORT=7275</item>
diff --git a/overlay/frameworks/base/core/res/res/values/config.xml b/overlay/frameworks/base/core/res/res/values/config.xml
index bc00ebc..dba7fa7 100644
--- a/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/overlay/frameworks/base/core/res/res/values/config.xml
@@ -26,7 +26,6 @@
         <item>XTRA_SERVER_1=https://xtrapath1.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_2=https://xtrapath2.izatcloud.net/xtra3grc.bin</item>
         <item>XTRA_SERVER_3=https://xtrapath3.izatcloud.net/xtra3grc.bin</item>
-        <item>NTP_SERVER=north-america.pool.ntp.org</item>
         <item>SUPL_ES=0</item>
         <item>SUPL_HOST=supl.google.com</item>
         <item>SUPL_PORT=7275</item>
-- 
2.17.1

